npc_simulator/src/node.cpp

```diff
@@ -1138,9 +1156,9 @@ dummy_perception_publisher::msg::Object NPCSimulator::convertObjectMsgToDummyPer
   output_obj.initial_state.twist_covariance.twist.angular.z = 0;
 
   if (obj->action == npc_simulator::msg::Object::ADD) {
-    //from second time, obj.action must change to MODIFY
+    // from second time, obj.action must change to MODIFY
     obj->action = npc_simulator::msg::Object::MODIFY;
-    sleep(0.01);  //avoid missing msg
+    sleep(0.01);  // avoid missing msg
```

Comment:
@yamacir-kit clang-tidy informs me that this function takes an int, so this is probably not working as intended. Could you comment on why the sleep is necessary here anyway, and give the correct duration to use?

Reply:
I removed this sleep since I don't see how it helps.

Reply:
@tkimura4 I hope that is fine.